src/js/NavigraphNavdataInterface.ts

```diff
@@ -0,0 +1,294 @@
+   */
+  public async set_active_database(path: string): Promise<void> {
+    return await this.callWasmFunction("SetActiveDatabase", { path })
+  }
+
+  public async get_database_info(ident: string): Promise<DatabaseInfo> {
+    return await this.callWasmFunction("GetDatabaseInfo", { ident })
+  }
+
+  public async get_airport(ident: string): Promise<Airport> {
+    return await this.callWasmFunction("GetAirport", { ident })
+  }
+
+  public async get_waypoints(ident: string): Promise<Waypoint[]> {
+    return await this.callWasmFunction("GetWaypoints", { ident })
+  }
+
+  public async get_vhf_navaids(ident: string): Promise<Waypoint[]> {
+    return await this.callWasmFunction("GetVhfNavaids", { ident })
+  }
+
+  public async get_ndb_navaids(ident: string): Promise<Waypoint[]> {
+    return await this.callWasmFunction("GetNdbNavaids", { ident })
+  }
+
+  public async get_airways(ident: string): Promise<Airway[]> {
+    return await this.callWasmFunction("GetAirways", { ident })
+  }
+
+  public async get_airways_at_fix(fix_ident: string, fix_icao_code: string): Promise<Airway[]> {
+    return await this.callWasmFunction("GetAirwaysAtFix", { fix_ident, fix_icao_code })
+  }
+
+  public async get_airports_in_range(center: Coordinates, range: NauticalMiles): Promise<Airport[]> {
+    return await this.callWasmFunction("GetAirportsInRange", { center, range })
+  }
+
+  public async get_waypoints_in_range(center: Coordinates, range: NauticalMiles): Promise<Waypoint[]> {
+    return await this.callWasmFunction("GetWaypointsInRange", { center, range })
+  }
+
+  public async get_vhf_navaids_in_range(center: Coordinates, range: NauticalMiles): Promise<VhfNavaid[]> {
+    return await this.callWasmFunction("GetVhfNavaidsInRange", { center, range })
+  }
+
+  public async get_ndb_navaids_in_range(center: Coordinates, range: NauticalMiles): Promise<Waypoint[]> {
+    return await this.callWasmFunction("GetNdbNavaidsInRange", { center, range })
+  }
+
+  public async get_airways_in_range(center: Coordinates, range: NauticalMiles): Promise<Airway[]> {
+    return await this.callWasmFunction("GetAirwaysInRange", { center, range })
+  }
+
+  public async get_controlled_airspaces_in_range(
+    center: Coordinates,
+    range: NauticalMiles,
+  ): Promise<ControlledAirspace[]> {
+    return await this.callWasmFunction("GetControlledAirspacesInRange", { center, range })
+  }
+
+  public async get_restrictive_airspaces_in_range(
+    center: Coordinates,
+    range: NauticalMiles,
+  ): Promise<RestrictiveAirspace[]> {
+    return await this.callWasmFunction("GetRestrictiveAirspacesInRange", { center, range })
+  }
+
+  public async get_communications_in_range(center: Coordinates, range: NauticalMiles): Promise<Communication[]> {
+    return await this.callWasmFunction("GetCommunicationsInRange", { center, range })
+  }
+
+  public async get_runways_at_airport(airport_ident: string): Promise<Waypoint[]> {
```

Comment:
Should this be a waypoint type?

Reply:
Yeah, I never picked any of these up because jest doesent have typings in expect calls... might consider switching to vitetest if that improves it